test/Reflection/conformance_descriptors.swift

```diff
@@ -5,7 +5,6 @@
 //
 // Temporarily disable on AArch64 Linux (rdar://88451721)
 // UNSUPPORTED: OS=linux-gnu && CPU=aarch64
```

Comment:
Do we still need this marked as unsupported on Linux?

Reply:
That one seems unrelated to this change.